app/services/clients/content_store.rb

```diff
@@ -3,9 +3,13 @@ class ContentStore
     class << self
       def find(path, attributes)
         response = HTTParty.get(end_point(path))
-        content_item = JSON.parse(response.body).symbolize_keys
-
-        content_item.slice(*attributes)
+        case response.code
+          when 200
+            content_item = JSON.parse(response.body).symbolize_keys
+            content_item.slice(*attributes)
+          when 404
+            false
```

Comment:
I would prefer to return `nil` here. It feels a [bit more idiomatic to Ruby](https://ruby-doc.org/core-2.2.3/Enumerable.html#method-i-find).